content/blog/Intro-to-HTML-CSS-and-JavaScript/index.md

```diff
@@ -0,0 +1,79 @@
+---
+{
+    title: "Introduction to HTML, CSS, and JavaScript",
+    description: 'Introduction to the underlying concepts of HTML, CSS, and JavaScript and how they work together.',
+    published: '2019-12-16T13:45:00.284Z',
+    authors: ['MDutro'],
+    tags: ['HTML', 'CSS', 'JavaScript', 'Beginner'],
+    attached: [],
+    license: 'publicdomain-zero-1'
+}
+---
+
+
+
```

Comment:
Let's remove the empty lines here. Let's only have a single empty line after the `---`